cfpb/cfgov-refresh
# Homepage: card styles

## Changes

- Add styles for cards in the footer area of the page.

## Testing

1. Pull branch, build and visit http://localhost:8000/?nhp=True
2. Scroll to bottom and resize window to view cards behavior.
3. Check that all is well on http://localhost:8000/ask-cfpb/

## Screenshots

<img width="1244" alt="Screen Shot 2019-11-29 at 10 19 16 AM" src="https://user-images.githubusercontent.com/704760/69969719-f398c680-14ea-11ea-943d-108ca06099ba.png">

<img width="575" alt="Screen Shot 2019-11-29 at 10 19 32 AM" src="https://user-images.githubusercontent.com/704760/69969720-f4315d00-14ea-11ea-8c19-f0fa136ee366.png">

<img width="588" alt="Screen Shot 2019-11-29 at 10 19 46 AM" src="https://user-images.githubusercontent.com/704760/69969721-f4315d00-14ea-11ea-9584-2d6c9735d75c.png">
